common/djangoapps/student/tests/test_reset_password.py

```diff
@@ -252,6 +252,25 @@ def test_reset_password_good_token(self):
         self.user = User.objects.get(pk=self.user.pk)
         self.assertTrue(self.user.is_active)
 
+    def test_password_reset_fail(self):
+        """Tests that if we providing mismatching passwords user is not marked as active"""
```

Comment:
Please correct the grammar here.  "if we providing" should be "if we provide".  So, the line can be:
"Tests that if we provide mismatched passwords, user is not marked as active."